reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -7974,10 +7974,30 @@ public final void subscribe(Subscriber<? super T> actual) {
 	 * <p>
 	 * It is advised to return a {@link Publisher} derived from the original {@link Flux}
 	 * in all cases, as not doing so would keep the original {@link Publisher} open and
-	 * hanging with a single request and no cancellation. Suppressing early errors while
-	 * maintaining this link can be achieved with {@link #onErrorResume(Function)} and
-	 * {@link #empty()}.
-	 *
+	 * hanging with a single request by the terminal signal of the returned
+	 * {@link Publisher}. Note, in such case only the terminal signal of the returned
+	 * {@link Publisher} or general stream cancellation will cancel the original
+	 * {@link Flux}.
+	 * <p>
+	 * Note, returning {@link Publisher} derived from the different source than original
```

Comment:
way too much notes and examples. I think the `Flux.just(1,2,3)` example should be introduced as a single commented line in the first snippet

Reply:
fixed